README.md

```diff
@@ -9,141 +9,173 @@ Please file a request in our
     ```
 1.  Run `install.sh` to install everything else needed to build this site:
 
-    ```
-    ./tool/install.sh
+    ```console
+    $ ./tool/install.sh
     ```
 
 > IMPORTANT:
 > - Any time you create a **new terminal/command window** to work on
 >   this repo, **repeat steps 1 and 2** above.
-> - If you upgrade Dart then rerun all of the steps above.
+> - If you upgrade Dart, then re-run all of the steps above.
 
-## Developing
+## Developing and serving changes
 
- 1. Create a branch.
- 1. Make your changes.
+ 1. After you clone this repo, create a branch. For example:
+ 
+    ```console
+    $ git checkout -b <BRANCH_NAME>
+    ```
+ 
+ 1. Make your changes, then commit them to the branch.
  1. Test your changes by serving the site locally.
-    Run either **one** of these commands:
-    - `./tool/serve.sh` (can also run via `npm run start`)
-
-    or
-    - `bundle exec jekyll serve --incremental --watch --livereload --port 4002`
-
-      **Note**: Unless you're editing files under `site-shared`, you can safely
-      ignore `ERROR: directory is already being watched` messages.
-      For details, see [#1363](https://github.com/flutter/website/issues/1363).
-
-      **Note**: The first time you run either one of these commands,
-      jekyll takes anywhere between 10 - 20 seconds to generate static
-      content inside the `_sites` directory. If you try to verify the
-      site locally but aren't able to see the content right away,
-      wait 20 seconds before stopping the
-      server or concluding that something is wrong.
- 1. Prior to submitting, validate site links:<br>
+    To serve locally, run _either one_ of these commands:
+    - ```console
+      $ ./tool/serve.sh # or npm run start)
+      ```
+
+    OR
+    - ```console
+      $ bundle exec jekyll serve --incremental --watch --livereload --port 4002
+      ```
+
+      > NOTE: Unless you're editing files under `site-shared`, you can safely
+      > ignore `ERROR: directory is already being watched` messages.
+      > For details, see [#1363](https://github.com/flutter/website/issues/1363).
+
+      > NOTE: The first time you run either one of these commands,
+      > Jekyll takes anywhere between 10-20 seconds to generate static
+      > content inside the `_sites` directory. If you try to verify the
+      > site locally, but aren't able to see the content right away,
+      > wait 20 seconds before stopping the
+      > server or concluding that something is wrong.
+ 1. Before submitting, validate site links:<br>
     `./tool/shared/check-links.sh`
 
 > TIP: Sometimes Jekyll gets confused and seems to be out-of-sync. (This might
 > happen, for example, when you pull from master and lots of files have moved.)
-> To fix Jekyll, stop the `serve.sh` script and remove the generated site files:
+> To fix Jekyll, stop the `serve.sh` script, remove the generated site files:
 > hand, and then restart the `serve.sh` script:
 
-> `npm run clean`
+> ```terminal
```

Comment:
I do something else. To remove the built files, I just run "jekyll clean" at the command line.